app/main.py

```diff
@@ -88,20 +91,21 @@ def process(self, encrypted_jwt, tx_id=None):
             decrypted_payload = self._decrypt(encrypted_jwt, tx_id)
 
             bound_logger.info("Extracting file")
-            decoded_contents, file_name, case_id = self.extract_file(decrypted_payload, tx_id)
+            decoded_contents, file_name, case_id, survey_id = self.extract_file(decrypted_payload, tx_id)
```

Comment:
returning this many variables from a method could be made more explicit and cleaner by using a Named Tuple.